scripts/install-cni.sh

```diff
@@ -30,6 +30,20 @@ cilium_ready() {
   # Send errors to stdout since they're "expected" errors.
   # This redirection doesn't affect exit status after execution.
   CNI_COMMAND=VERSION /host/home/kubernetes/bin/cilium-cni 2>&1
+
+  # Wait upto 60s for the cilium pod to report healthy.
+  echo "Waiting for upto 60s for cilium to report readiness"
+  timeout 60s \
+    bash -c \
+      'HEALTHZ_PORT="${CILIUM_HEALTHZ_PORT:-9879}" ; \
+      echo "Querying cilium healthz on port ${HEALTHZ_PORT}" ; \
+      while ! curl -fsSm 1 -o /dev/null http://localhost:${HEALTHZ_PORT}/healthz; do \
```

Comment:
Let's move the stderr output to stdout, so it doesn't go to "ERRORS" in logs confusing users. Either do it here or on the whole timeout call.